[441352] Add support for polygon-based node shapes

miklossy opened the pull request:
- Add rendering support for "ellipse", "invtriangle" and "triangle" node
  shapes.

Signed-off-by: Tamas Miklossy miklossy@itemis.de
Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=441352

nyssen commented:
Please consider https://bugs.eclipse.org/bugs/show_bug.cgi?id=328801#c12, where I have argued why this proposed fix is inappropriate (and thus cannot be pulled in).

miklossy committed:
[441352] Add support for polygon-based node shapes

- Add rendering support for "ellipse", "invtriangle" and "triangle" node
shapes.

Signed-off-by: Tamas Miklossy <miklossy@itemis.de>
Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=441352
